docs/UserGuide.md

```diff
@@ -326,17 +326,17 @@ If you wish to generate a set of sample endpoints, you may delete the **imposter
 
 **Description:** [Call](#glossary-call) an API endpoint directly (without saving) **(an ongoing call can be cancelled with <kbd>ctrl</kbd> + <kbd>d</kbd>)**.
 
-**Format:** <span class="main-command">run</span> <span class="optional-param">-x METHOD</span> <span class="optional-param">-u URL</span> <span class="optional-param">-d DATA</span> <span class="optional-param">[-h HEADER]</span>
+**Format:** <span class="main-command">run</span> <span class="compulsory-param">-x METHOD</span> <span class="compulsory-param">-u URL</span> <span class="optional-param">-d DATA</span> <span class="optional-param">[-h HEADER]</span>
 
-**Example & Output:** <span class="main-command">run</span> <span class="optional-param">-x GET</span> <span class="optional-param">-u https://api.data.gov.sg/v1/environment/pm25</span>
+**Example & Output:** <span class="main-command">run</span> <span class="compulsory-param">-x GET</span> <span class="compulsory-param">-u https://api.data.gov.sg/v1/environment/pm25</span>
 
 <p align="center">
   <img width="450px" src="images/commands/run.png" >
 </p>
 
 <div markdown="span" class="alert alert-warning">:bulb: **Tip:**
-A shorthand for <span class="optional-param">GET</span> requests can be done without specifying <span class="optional-param">-x</span> and <span class="optional-param">-u</span> 
-(for example: <span class="main-command">run</span> <span class="optional-param">https://api.data.gov.sg/v1/environment/pm25</span>).
+A shorthand for <span class="optional-param">GET</span> requests can be done without specifying <span class="compulsory-param">-x METHOD</span> <span class="compulsory-param">-u URL</span> 
```

Comment:
For the case of a shorthand would these 2 fields better be reflected as optional? It was raised as a bug report when highlighted as compulsory previously.

Reply:
I think we should treat the shorthand syntax just as a unique syntax. This is because if u say it is optional, then entries such as 'run -x get ' and 'run -u https://test.com' might be seen as valid.

Reply:
Also I believe the inconsistency u are referring to is #518, if u look closely, it is not about that but about the table showing different format.

Reply:
I think the issue for this is #526, in which the user brought up a valid point that reading the tip as it is makes the highlighting seem mismatching (since the compulsory parameters did not appear in the example that immediately followed it). I think arguments can be made both ways so safest option might be to split things up like in the `find` command although the difference for `run` is not as complex.

Reply:
Good point, I have updated the UG such that the format is explained separately.